src/config/__test__/validate.test.ts

```diff
@@ -1,29 +1,62 @@
 import { validateConfig } from '../validate.js'
 import test from 'ava'
 import { RawConfig } from '../types.js'
+import { InvalidUrl } from '../../errors/invalidUrl.js'
+import { config } from '../index.js'
+import { InvalidConfigError } from '../../errors/invalidConfig.js'
 
-const config: RawConfig = {
-	port: 'mock-port',
-	appName: 'mini-test',
-	baseRedirectUrl: 'http://mock.com',
-	url: {
-		matchPattern: 'mock-patter',
-		lifetime: '120',
-	},
-	storage: {
-		driverName: 'InMemory',
-		relationalDriverConfig: {
-			client: 'mock-client',
-			connection: {
-				host: 'local-host',
-				user: 'snir-hamair',
-				password: 'admin1234',
-				database: 'mock-db',
+function getConfig(): RawConfig {
+	return {
+		port: 'mock-port',
+		appName: 'mini-test',
+		baseRedirectUrl: 'http://mock.com',
+		url: {
+			matchPattern: 'mock-patter',
+			lifetime: '120',
+		},
+		storage: {
+			driverName: 'InMemory',
+			relationalDriverConfig: {
+				client: 'mock-client',
+				connection: {
+					host: 'local-host',
+					user: 'snir-hamair',
+					password: 'admin1234',
+					database: 'mock-db',
+				},
 			},
 		},
-	},
+	}
 }
-
 test('validate config - happy flow [configuration set propely]', (t) => {
+	t.is(validateConfig(getConfig()), true)
+})
+
+test('Making sure that validateConfig throws if an invalid baseRedirectUrl is given', (t) => {
+	const invalidUrls = [
+		{ url: 'mock.com', message: 'Throws when the baseRedirectUrl has no protocol' }, //test passes
+		{ url: 'blabla...', message: 'Throws when the baseRedirectUrl is not valid' }, //test passes
```

Comment:
comments are redundant.
maybe we can also add a case for having a protocol in bad format (e.g `htts` or some typo)

Reply:
I agree about the comments, you can remove them.

About the protocol, `htts://google.com` is actually a valid URL, but assuming we only send HTTP requests, we should check it (along with the port) in validateConfig. That being said, it's not a part of **this** PR, since this PR is about tests.